demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1581,6 +1599,14 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
   contentShareDidUnpause(): void {
     this.log(`content share unpaused.`);
   }
+
+  encodingSimulcastLayerDidChange(simulcastLayers: SimulcastLayers): void {
+    this.log(`current active simulcast layers changed to: ${SimulcastLayerMapping[simulcastLayers]}`);
+  }
+
+  remoteVideosAvailableDidChange(attendees: { attendeeId: string; externalUserId: string }[]) {
```

Comment:
I will defer to the core SDK team on the preference here.  Both seem reasonable.
That being said, I do think the current naming is descriptive.
When originally naming this I was thinking about correlating with the current naming conventions of video tiles.  The idea being a builder would be using this new API and comparing against current video tiles.
Some of the current API names I was using as a model are getAllRemoteVideoTiles and remoteVideoTilesByAttendeeId

Reply:
We also need to have some plural form here, since we will get the array of remote attendee videos. `remoteVideosAvailableDidChange` or `remoteVideosAvailablityDidChange`. But yes will wait for feedback.